README.md

```diff
@@ -39,6 +39,12 @@ The [Bot](bot/bot.go) holds references to structures that we use to communicate
 
 [server.go](server/server.go) defines some handlers for HTTP requests. Our website will make requests to `/api/teams` and `/api/members` to display information about our teams and members. Note that content is served over HTTPS using `acme/autocert` to get TLS certificates from LetsEncrypt.
 
+#### Plugins
+
+A Rocket Plugin is simply any type that implements the [Plugin](plugin/plugin.go) interface. A plugin is intended to be a standalone component of Rocket. The prime example of a plugin is Rocket's [Core](core/core.go) plugin which provides basic Launch Pad administration commands for managing teams and users on both Slack and GitHub. You can use the `Start` method of your plugin start any background tasks you need to. Any `Commands` and `EventHandlers` you expose to Rocket in your implementation of the Plugin interface will be automatically registered with the `Bot`. See the Slack's [API Event Types](https://api.slack.com/events) for a list of events and their names if you implement your own `EventHandler`s for your plugin.
```

Comment:
> You can use the `Start` method of your plugin **to** start any background tasks...

Also, I feel like some more paragraph separation here would help with readability, as well as simplified code blocks to demonstrate what the plugins roughly look like without digging too deep into the source code

Reply:
I think this just looks longer than it really is because of the thin margin. I think it's pretty readable, and it's about as long as some of the other paragraphs in here.
![image](https://user-images.githubusercontent.com/9647946/40219290-95ce414c-5a29-11e8-9f41-d8cf0ab42e27.png)

I added `welcome.go` as another more simple example of a plugin because I felt like trying to put a whole plugin into the README as a code snippet would be too much.